Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -3,14 +3,17 @@ import Combine
 import SwiftUI
 import XCTestDynamicOverlay
 
-// MARK: - Deprecated after 0.50.4
-
-@available(
-  *,
-  deprecated,
-  message: "Use 'EffectTask<Action>.Send' instead."
-)
-public typealias Send<Action> = EffectTask<Action>.Send
+// MARK: - Deprecated after 0.51.0
+
+// NB Soft deprecation to avoid deprecation messages when this type is inferred in `Effect`
+// without having to qualify it as `ComposableArchitecture.Send` there.
+@available(iOS, deprecated: 9999.0, message: "Use 'Send<Action>' instead.")
+@available(macOS, deprecated: 9999.0, message: "Use 'Send<Action>' instead.")
+@available(tvOS, deprecated: 9999.0, message: "Use 'Send<Action>' instead.")
+@available(watchOS, deprecated: 9999.0, message: "Use 'Send<Action>' instead.")
+extension EffectTask {
+  public typealias Send = ComposableArchitecture.Send
+}
```

Comment:
This reinstalls a soft-deprecated `EffectTask.Send` in any case someone relied on it in a context where `Action` was furthermore inferred.
This can probably be removed.